Documentation/slip-rate-on-DR.rst

```diff
@@ -0,0 +1,17 @@
+Slip-rate imposed on a DR boundary
+===================================
+
+This "pseudo" friction law allows imposing slip-rate on a dynamic rupture boundary.
+The FL id for this friction law is 33.
+The advantage of this approach compared to a multi point-source representation is that the fault slip is not condensed to points. 
+Therefore the discontinuity of the displacement across the fault can be accurately accounted for.
+In addition, no spurious waves, related to the multi point-sources representation, are generated.
```

Comment:
I'd reformulate the above a bit (just a proposal):

"The advantage of this approach is that the displacement discontinuity can be represented in SeisSol's discontinuous finite element space. Therefore, no spurious waves arise. A multi point-source representation, in comparison, may give rise to spurious waves due the continuity of the basis functions within a finite element."

Reply:
I think the main issue is that fault slip is condensed to points. 
Even if Seissol could perfectly account for point sources (without smearing due to projection to the basis function), this would still be a problem.

Reply:
What about:
"The advantage of this approach is that the displacement discontinuity can be ACCURATELY represented in SeisSol's discontinuous finite element space.  A multi point-source representation, in comparison, may give rise to spurious waves due the continuity of the basis functions within a finite element (smearing), and  to the point-wise representation itself (aliasing)."

Reply:
Sounds good but I don't agree with the statement that point sources are a problem per se. If you have a point source, e.g. in the  homogeneous full space, and analytically solve the problem then you don't get "spurious waves". (See e.g. Green's function formula in Aki & Richards book.)

Reply:
> You can also have scripts as part of a pip module

ok then let's do that in a separate PR.

Reply:
Are we talking about this part of the sentence:
`and to the point-wise representation itself (aliasing).`?
Here I m talking of the discretization of the rupture in multiple point sources.
For example a rupture front of duration T propagating at constant speed Vr cannot be properly captured by multiple point sources if the sampling h >> Vr/T.